ui/src/components/songsView/SongsButtonsCard.tsx

```diff
@@ -0,0 +1,182 @@
+import { flatMusicKeysOptions, sharpMusicKeysOptions } from '../../constants';
+
+type SongsButtonCardProps = {
+  song: SongViewSchema | undefined;
+};
+
+const SongsButtonsCard = (props: SongsButtonCardProps) => {
+  const song = props.song;
+  const [chordStatus, setChordStatus] = useState(false);
+  const [useFlat, setUseFlat] = useState(false);
+  const [count, setCount] = useState(sharpMusicKeysOptions.indexOf(song?.originalKey ?? 'C'));
+  const [split, setSplit] = useState(1);
+  const isDesktop = useMediaQuery('(min-width:768px)');
+
+  const handleChange = (setter: React.Dispatch<React.SetStateAction<boolean>>) => {
+    return (event: React.ChangeEvent<{}>, value: boolean) => {
+      setter(value);
+    };
+  };
+
+  const handleIncrement = () => {
+    if (count < 11) setCount(count + 1);
+    else setCount(0);
+  };
+
+  const handleDecrement = () => {
+    if (count > 0) setCount(count - 1);
+    else setCount(11);
+  };
+
+  const handleSplit = () => {
+    if (split < 3) setSplit(split + 1);
+    else setSplit(1);
+  };
+
+  useEffect(() => {
+    setCount(sharpMusicKeysOptions.indexOf(song?.originalKey ?? 'C'));
+  }, [song]);
+
+  return (
+    <>
+      <Container>
+        <Box sx={{ bgcolor: 'white', p: 2 }}>
+          <Box
+            sx={{
+              bgcolor: '#FAFAFA',
+              boxShadow: 0,
+              borderRadius: 0,
+              p: 0,
+              width: '100%',
+              padding: '12px',
+            }}
+          >
+            <Stack direction="row" spacing={{ base: '0', xs: '1vh', sm: '1.5vh', md: '2vh' }}>
+              <Box justifyContent="left" bgcolor="white" padding="4px 12px" borderRadius="4px">
+                <FormGroup>
+                  <FormControlLabel
+                    labelPlacement="start"
+                    control={
+                      <Switch
+                        checked={chordStatus}
+                        onChange={handleChange(setChordStatus)}
+                        name="chord"
+                      />
+                    }
+                    label="Chord"
+                  />
+                </FormGroup>
+              </Box>
+              <Stack
+                direction="row"
+                spacing={{ base: '0', xs: '0', sm: '1px', md: '2px' }}
+                bgcolor="white"
+                padding="4px 12px"
+                borderRadius="4px"
+              >
+                <Box bgcolor={'white'}>
+                  <Typography alignItems={'center'} paddingTop="5px">
+                    Key
+                  </Typography>
+                </Box>
+                <Box bgcolor="primary.lightest">
+                  <IconButton color="primary" aria-label="down" onClick={handleDecrement}>
+                    <KeyboardArrowDownIcon />
+                  </IconButton>
+                </Box>
+                <Chip
+                  label={useFlat ? flatMusicKeysOptions[count] : sharpMusicKeysOptions[count]}
```

Comment:
<img width="135" alt="image" src="https://github.com/hmcc-global/psalted-2/assets/24607458/1b2ee7a9-3733-489f-ba60-35e0d1e536bb">

looks a bit odd on mobile, think can add breakpoints width for the increment and decrement button and also vertically align the chip